portfolio/src/main/webapp/projects.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Lorraine Lyu's Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <div id="content">
+      <ul id="navbar">
+        <li class="nav-button"><a href="index.html" class="nav-link">Home</a></li>
+        <li class="nav-button"><a href="introduction.html" class="nav-link">Introduction</a></li>
+        <li class="nav-button current"><a class="nav-link">Projects</a></li>
+        <li class="nav-button"><a href="gallery.html" class="nav-link">Gallery</a></li>
+      </ul>
```

Comment:
Since the navbar is always the same and on every page, I recommend extracting it out to its own file and then adding the contents. This way you only have to make changes once. Feel free to make a bug and a TODO and do this in a follow up PR.